ddtrace/internal/remoteconfig/worker.py

```diff
@@ -174,6 +174,7 @@ def __enter__(self):
 
     def __exit__(self, *args):
         # type: (...) -> None
+        self.stop_subscribers(True)
```

Comment:
Joining on process shutdown is likely to cause a delay, so I'd join here only if we really have to, to ensure a proper state throughout. The impact is most noticeable for applications that spawn short-lived processes, in which we might end up adding some latency.

Reply:
`self.stop_subscribers(True)` is needed, if not, the test fails. `__exit__` is a function that we use in tests only. :+1: